custom_components/elegoo_printer/image.py

```
@@ -0,0 +1,71 @@
+        super(ElegooPrinterEntity, self).__init__(coordinator=coordinator)
+        self.coordinator = coordinator
+        self._attr_content_type = "image/bmp"
+        self._image_filename = None
+        self.entity_description = description
+        self._attr_unique_id = self.entity_description.key
+        self._attr_image_url = "http://10.0.0.212:3030/media/mmcblk0p1/history_image/6c4336e6-eecb-11ef-ae89-40f4c926a324.bmp"
+
+    # @property
+    # def image(self) -> bytes | None:
+    #     """Return bytes of an image."""
+    #     self._attr_content_type = "image/bmp"
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)